modules/claims_api/app/swagger/claims_api/claims_v0_controller_swagger.rb

```diff
@@ -17,14 +20,7 @@ class ClaimsV0ControllerSwagger
           key :name, :id
           key :in, :path
           key :description, 'The ID of the claim being requested'
-          key :required, true
-          key :type, :string
-        end
-
-        parameter do
-          key :name, 'apikey'
-          key :in, :header
-          key :description, 'API Key given to access data'
+          key :example, ''
```

Comment:
All of these examples are great 👍

Reply:
So we're going to make a separate issue for adding examples to all of them, for now I'll remove the blank ones.

I started to do it, but it was pretty daunting